src/Network.ts

```diff
@@ -244,9 +244,7 @@ export class Network {
             name: this.name,
             chainId: this.chainId,
             gateway: this.gateway.address,
-            AxelarGasService: {
-              address: this.gasReceiver.address
-            },
+            gasReceiver: this.gasReceiver.address,
```

Comment:
Prefer to use the future proof schema, `gasReceiver` is a deprecated field now. Perhaps rename it to `gasService` to better match the contract name?

Reply:
Agreed. we can replace `gasReceiver` with `gasService` everywhere, but it still depends on the value from https://github.com/axelarnetwork/axelar-cgp-solidity/blob/3e7d6751a3016e4694b1ce643f11530442731e09/info/testnet.json#L10

Probably need to update there first, then we can update it here.